src/store/index.js

```diff
@@ -0,0 +1,48 @@
+import {combineReducers, createStore, applyMiddleware} from 'redux';
+import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
+
+import safeStorage from './utils/safe_storage';
+
+// Reducers
+import user from './user_reducer';
+
+// Root reducer
+const rootReducer = combineReducers({
+  user,
+});
+
+// Middlewares
+const middlewares = [thunk];
+
+if (process.env.NODE_ENV !== 'production') {
+  const logger = createLogger();
+  middlewares.push(logger);
+}
+
+// Persisted state
+const persistedState = {
+  user: {
+    isAuthenticated: false,
+    token: safeStorage.get('token') || null,
+  },
+};
+
+// Store
+const store = createStore(
+  rootReducer,
+  persistedState,
+  applyMiddleware(...middlewares),
+);
+
+// Persist change to local storage
+store.subscribe(() => {
```

Comment:
@ajithranka I was wondering if we are going to save more things to localStorage in the future?